packages/block-library/src/table/transforms.js

```diff
@@ -1,15 +1,27 @@
+/**
+ * WordPress dependencies
+ */
+import { createBlock, getBlockAttributes } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import { name } from './block.json';
+import { getListContentSchema } from '../list/transforms';
+import { getListItemBullet } from './utils';
+
 const tableContentPasteSchema = ( { phrasingContentSchema } ) => ( {
 	tr: {
 		allowEmpty: true,
 		children: {
 			th: {
 				allowEmpty: true,
-				children: phrasingContentSchema,
+				children: getListContentSchema( { phrasingContentSchema } ),
```

Comment:
I'm not sure I like this. Why not use `'*'` for `children` and let raw handling parse the cell contents as blocks, then transform that to the markdown syntax. OR instead of parsing as blocks, just convert the HTML to mark down?

https://github.com/WordPress/gutenberg/blob/65170083fb9e69620968765ad19bf6d258b12a12/packages/block-library/src/quote/transforms.js#L38-L61

```suggestion
				children: '*',
```

Reply:
Although, when it's parsed as block, it's harder to convert all blocks to something textual (like heading to plain text etc.)

Reply:
Why don't we build this into the paste handler and generally convert lists to plain text if the allowed children is `phrasingContentSchema`? This behaviour could be useful in other places. What if there's a list in a caption? Generalising this would be great.

So inside the paste handler, I think we need a filter before `removeInvalidHTML` that converts list that are not allowed in these places (not in schema) to the text version of the list.

Let me know if you need help here.